redirects.yaml

```diff
@@ -569,6 +569,7 @@ server/get-ubuntu/cd-and-dvd/?: "https://shop.canonical.com/index.php?cPath=17"
 server/get-ubuntu/cds/?: "https://shop.canonical.com/index.php?cPath=17"
 server/get-ubuntu/download/?: "/download/server"
 server/how-can-it-be-free/?: "/server"
+server/livepatch/?: "/support/livepatch"
```

Comment:
There are some links to `/server/livepatch/` throughout the site. These should be updated too.